### PR TITLE
cmd/plume: Add "dev" channel for testing release process

### PR DESCRIPTION
**Pull request by bgilbert**

### CODE REVIEW

File: cmd/plume/specs.go [hunk 1]
```diff
@@ -310,7 +373,7 @@ func AddSpecFlags(flags *pflag.FlagSet) {
 	flags.StringVarP(&specBoard, "board", "B",
 		board, "target board")
 	flags.StringVarP(&specChannel, "channel", "C",
-		"alpha", "channels: "+channels)
+		"dev", "channels: "+channels)
```
Comment (euank) [inline cmd/plume/specs.go]: perhaps this should just default to "" and be required instead.

File: cmd/plume/specs.go [hunk 2]
```diff
@@ -152,6 +156,56 @@ var (
...
+			},
+			AWS: awsSpec{
+				BaseName:        "ContainerLinuxDev",
+				BaseDescription: "CoreOS Container Linux development image",
+				Prefix:          "coreos_production_ami_",
+				Image:           "coreos_production_ami_vmdk_image.vmdk.bz2",
+				Partitions: []awsPartitionSpec{
+					awsPartitionSpec{
+						Name:         "AWS West",
+						Profile:      "default",
```
Comment (euank) [inline cmd/plume/specs.go]: I think requiring the name to be `coroes-dev` is slightly safer, but I don't feel strongly.

File: cmd/plume/specs.go (hunk 1 above)
Comment (bgilbert) [inline cmd/plume/specs.go]: It's convenient to not have to specify `dev` when developing.

File: cmd/plume/specs.go (hunk 2 above)
Comment (bgilbert) [inline cmd/plume/specs.go]: Updated.

**Pull request merged**